### PR TITLE
feat(DB): Set faction leaders and capital city guards to their vanilla stats

### PR DESCRIPTION
**Pull request by JanEbbing**
This was a suggestion for chromiecraft as it will encourage world pvp.
This diff will have 2 parts: Changing the faction leaders, and changing all the guards in the capital cities to be lvl 60.
Ideally, this diff would be behind some configuration flag so that any server could choose whether or not to use the vanilla or wotlk values.
I used wowhead to get their classic HP values, e.g. https://classic.wowhead.com/npc=4949/thrall . For faction leaders added in TBC I just scaled them equally.
I tried testing the level, as it just shows ?? Boss and I didn't find a GM command to show the level, I made my character level 60, learned all weapon skills and hit an unedited faction leader (lv 83) and a modified one, but I didn't even miss on the 83 one. I thought a level 60 was supposed to miss basically every hit on a 83 boss?
I did not verify yet if their abilities are dealing lv 80 damage or lv 60.

To get the IDs I did the following: Searched creature_template by name, verified by using .go creature <ID> command that it's the right NPCs, adjusted the values, then spawned the faction leaders to check their values.

## Changes Proposed:
-  Sets all faction leaders to lvl 63 and their vanilla HP values in order to give another world PvP objective on chromie
-  In next commit: Set all guards in capital cities to level 63

## Issues Addressed:

-

## Tests Performed:

- DB change only. Tested by spawning them in-game (to avoid teleporting 10 times). Verified that the IDs are correct by teleporting to each one beforehand.
grafik

Sample screenshot of test, I have one for each faction leader (ignore the changed name of Varian, this is my first PR and that was my first change :) )
WoWScrnShot_071921_031034

## How to Test the Changes:

1. Apply SQL changes
2. Restart world server
3. TP to any of the creatures from the table above, verify that they have 60 stats.

## Known Issues and TODO List:

- [TODO] Verify that their combat strength is appropriate for 60s
- [TODO] Change guards in capital cities to be 62 as well (currently: 82)

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_characters/rev_1626661382000836600.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_characters` (`sql_rev`) VALUES ('1626661382000836600');
+
+
+UPDATE acore_world.creature_template SET minlevel = 63,maxlevel=63,HealthModifier=180.882  where entry IN (4949, 10181, 10540, 3057, 16802, 29611, 2784, 7937, 7999, 17468)
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_characters/rev_1626661382000836600.sql:4]: Hello an thx for ylur contribution! Please read https://www.azerothcore.org/wiki/sql-standards

- No need to call DB Name here
- Add backticks ` around column names 
- Write keywords in uppercase
- Semicolon is missing at the end

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_characters/rev_1626661382000836600.sql (hunk 1 above)
Comment (JanEbbing) [reply data/sql/updates/pending_db_characters/rev_1626661382000836600.sql:4]: Hey, sorry what do you mean by "No need to call DB Name here"? Putting the DB name into a variable?

**Comment by JanEbbing** [comment]
Fixed all comments.

**Review by LukasVolgger — APPROVED** [review]
Now it's fine, thx! :)

**Comment by JanEbbing** [comment]
Verified that faction leaders still deal lvl 80 damage with melee and presumably abilities. Working on a bigger PR that fixes all of that.

**Comment by JanEbbing** [comment]
Updated to change all city and royal guards + special NPCs in capitals to have their vanilla stats.
Playtested in all capital cities, sample screenshots:

WoWScrnShot_071921_160457

WoWScrnShot_071921_155538

**Comment by FrancescoBorzi** [comment]
are they supposed to be like this in 3.3.5a?

if yes, then this is good to go

if not, then this can be moved to https://github.com/azerothcore/progression-system

**Comment by JanEbbing** [comment]
@FrancescoBorzi No, current values are correct for 3.3.5a, this is just for the brackets up to 60. Should I open a PR in the other repo?

**Comment by FrancescoBorzi** [comment]
> @FrancescoBorzi No, current values are correct for 3.3.5a, this is just for the brackets up to 60. Should I open a PR in the other repo?

yes please, I'll close this one.

Basically on this repo we only place things that are blizzlike 3.3.5a (unless you can introduce something optional and disabled by default in conf or similar). Then for the progression we use that other repo.

**Pull request closed without merging**